ansible/README.md

```diff
@@ -102,15 +102,18 @@ Yes, in order to access the package repositories (we will perform either `yum in
 As Ansible can't run on Windows, it has to be run on a seperate system and then pointed at a Windows machine (such as a VM). To test the playbook, a vagrant VM can be used.
 You can do this by following these steps:
 
-1) Run `vagrant plugin install vagrant-disksize`. This will install a plugin that allows the user to specify the disksize of the VM.
+1) Run `vagrant plugin install vagrant-disksize`. This will install a plugin that allows the user to specify the disksize of the VM and is required to use the Windows Vagrantfile.
 2) `git clone "https://github.com/AdoptOpenJdk/openjdk-infrastructure/"`
-3) `ln -sf Vagrantfile.WindowsS12 Vagrantfile && vagrant up` in the cloned `/openjdk-infrastructure/ansible` directory
-4) Note the IP address that is output from running `vagrant up` and place it into a text file. e.g. `hosts.win`. Place this text file in `../anisble/playbooks/AdoptopenJDK_Windows_Playbook`
+3) `ln -sf Vagrantfile.Win2012 Vagrantfile && vagrant up` in the cloned `/openjdk-infrastructure/ansible` directory
+3.5) You can edit `Vagantfile.Win2012` so the GUI is shown when running `vagrant up` by changing `v.gui = false` to `v.gui = true`
```

Comment:
I would say "You can optionally add" then "but in most cases this isn't necessary" to the end. Also please re-number so this is step 4 ;-)